apps/files_versions/lib/AppInfo/Application.php

```diff
@@ -125,7 +130,7 @@ private function loadBackend(array $backend, ContainerInterface $container, ILog
 			$backendObject = $container->get($class);
 			$versionManager->registerBackend($for, $backendObject);
 		} catch (\Exception $e) {
-			$logger->logException($e);
+			$logger->critical($e);
```

Comment:
```suggestion
			$logger->errpr($->getMessage(), ['exception' => $e]);
```

then it's equivalent to `logException($e)`